### ISSUE TITLE
Site Designs WIP

### ISSUE THREAD

**Issue by richardbirley** [leicesterjs/site#53]
I have created some preliminary designs needs work and dev contributors to help get going.

I can provide graphics need coding into the site... volunteers welcome!

**Comment by richardbirley** [comment]
![image (1)](https://user-images.githubusercontent.com/15819101/68309927-1c25d000-00a7-11ea-8434-cb1b5e4f7245.png)

**Comment by joshghent** [comment]
Like the design mate but think it should be a bit more content rich and make the orange bolder. What does everyone else think?
Let's get this design finalized before we put pen to paper so to speak.

**Comment by Adam-STaC-Solutions** [comment]
Initial thoughts are positive! Would be good to see where the content fits in.

**Comment by richardbirley** [comment]
Yes these designs are no where near finished, this was like 30 mins work.. needs more thought with the content, and structure...

We need to talk about the Information Architecture and what info is displayed on what pages etc..

If people are happy to help with content and what we feature.. I will happily put time in on the graphics, the Orange is no problem Josh just an opacity tweak :)

**Comment by richardbirley** [comment]
Was thinking we can do a live counter to the next meetup
like a ticking clock if it isn't to expensive from dev perspective, or even a daily countdown would suffice!

**Comment by Adam-STaC-Solutions** [comment]
Happy to assist. Countdown FTW!

**Comment by Sehbaz** [comment]
Awesome design 🔥

How this design is going to respond on mobile devices, just the initial prototype would be better to acknowledge web flow.

**Comment by richardbirley** [comment]
Hi these are throw away visuals right now.. nothing to commit to, I wanted to show a style.. but would wireframe the concepts on mobile first and then see how that translates to desktop.
